src/components/NewCardForm.js

```diff
@@ -7,25 +7,28 @@ const NewCardForm = ({ boardId, createNewBoard }) => {
   const [message, setMessage] = useState("");
   const [invalidForm, setInvalidForm] = useState(true);
 
-  const generateMessage = (e) => {
-    setMessage(e.target.value)
-    if (e.target.value.length > 0 && e.target.value.length <= 40) {
+  const generateMessage = (event) => {
+    setMessage(event.target.value)
+    if (event.target.value.length > 0 && event.target.value.length <= 40) {
       setInvalidForm(false);
     } else {
       setInvalidForm(true);
     }
   }
 
-  const handleSubmit = (e) => {
-    e.preventDefault();
+  const handleSubmit = (event) => {
+    event.preventDefault();
     console.log(message);
+    console.log(boardId)
```

Comment:
We'll probably want to get rid of these console logs before we turn in the project, but everything else looks good!

Reply:
lol yea I just noticed them - sorry about that!